plugins/notifications/src/components/NotificationsFilters/NotificationsFilters.tsx

```diff
@@ -224,6 +248,25 @@ export const NotificationsFilters = ({
             </Select>
           </FormControl>
         </Grid>
+
+        <Grid item xs={12}>
+          <FormControl fullWidth variant="outlined" size="small">
+            <InputLabel id="notifications-filter-severity">Severity</InputLabel>
+
+            <Select
+              label="Minimal severity"
```

Comment:
Not a native speaker but somehow I think `Minimum severity` would sound better 🤷

Reply:
Just googled it and `minimum` is correct in this context :-)